ssr/webpack.config.js

```diff
@@ -8,7 +8,8 @@ module.exports = {
   entry: "./index.js",
   output: {
     path: distPath,
-    filename: "cli.js"
+    filename: "ssr.js",
+    libraryTarget: "commonjs2"
```

Comment:
Why/what is this?

Reply:
Otherwise the ssr module can't be imported by our cli module. Webpack doesn't create importable bundles by default.